### ISSUE TITLE
Error when importing in tests

### ISSUE THREAD

**Issue by nosovsh** [pradel/react-minimalist-portal#17]
When library is used in node environment (e.g. in tests) you get the following error:
```
ReferenceError: Element is not defined
    at Object.<anonymous> (....../node_modules/react-minimalist-portal/lib/portal.js:89:45)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
```
I'm using mocha to run tests so I can quick fix it with mocking `global.Element = null;` but not sure it is the best approach.

**Comment by pradel** [comment]
I think you will nee to use [jsdom](https://github.com/jsdom/jsdom) in order to get the tests running correctly

**Comment by pradel** [comment]
You can take a look at this file https://github.com/pradel/react-minimalist-portal/blob/master/test/setup.js#L8 It's how I setup jsdom for mocha.

**Comment by markzeras** [comment]
Started getting the same error here :/
``` 
ReferenceError: Element is not defined
    at Object../react/node_modules/react-minimalist-portal/lib/portal.js
```

**Comment by leticiawanderley** [comment]
Me too, while using the [react-responsive-modal](https://github.com/pradel/react-responsive-modal).

**Comment by pradel** [comment]
closed in 2.3.1 thanks to @leticiawanderley
